doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+:banner: banners/iap.jpg
+:types: api
+
+:code-column:
+
+===============
+Extract API
+===============
+
+Odoo provide a service allowing you to automate the processing of your invoices. The service scan your document using an Optical Character Recognition engine (OCR) and then use AI based algorithms to extract the fieds of interest as the total, the due date or even the invoice lines. More information can be found on the `demo page <https://www.odoo.com/page/invoice-automation>`_.
+
+This service is a paid service. Each invoice processing will cost you one credit. Three packs of different size can be bought on `iap.odoo.com <https://iap.odoo.com/iap/in-app-services/259?sortby=date>`_.
+
+You can either use this service directly in the Odoo Accounting App or by using the API. The Extract API which is detailed in the next section allows you to integrate our service directly in your own projects.
+
+Invoices
+========
+The extract API can be contacted at the following JSON-RPC2_ endpoint: `https://iap-extract.odoo.com`
+
+Expected successful flow
+------------------------
+
+1. Call :ref:`/iap/invoice_extract/parse <webservices/extract_api/invoice_parse>` to submit your invoices (one call for each invoice). On success you receive a `document_id` in the response. 
+2. You then have to regularly poll the :ref:`/iap/invoice_extract/get_results <webservices/extract_api/invoice_get_results>` route to get the document's parsing status.
+3. Once the result received you can validate it by calling :ref:`/iap/invoice_extract/validate <webservices/extract_api/invoice_validate>` and sending the expected values. This step is optional but greatly helps the system to improve.
```

Comment:
(Comas) 
1. Call :ref:`/iap/invoice_extract/parse <webservices/extract_api/invoice_parse>` to submit your invoices (one call for each invoice). On success**,** you receive a `document_id` in the response. 
2. You then have to regularly poll the :ref:`/iap/invoice_extract/get_results <webservices/extract_api/invoice_get_results>` route to get the document's parsing status.
3. Once the result received**,** you can validate it by calling :ref:`/iap/invoice_extract/validate <webservices/extract_api/invoice_validate>` and sending the expected values. This step is optional but greatly helps the system to improve.